modules/nlu/src/backend/prediction-handler.test.ts

```diff
@@ -23,17 +24,19 @@ const loggerMock = (<Partial<Logger>>{
 }) as Logger
 
 const makeModelsByLang = (langs: string[]) => {
-  const models: NLU.ModelId[] = (<Partial<NLU.ModelId>[]>langs.map(l => ({ languageCode: l }))) as NLU.ModelId[]
+  const models: nluCore.ModelId[] = (<Partial<nluCore.ModelId>[]>(
+    langs.map(l => ({ languageCode: l }))
+  )) as nluCore.ModelId[]
   return _.zipObject(langs, models)
 }
 
-function makeEngineMock(loadedLanguages: string[]): NLU.Engine {
-  const loadedModels: NLU.ModelId[] = (<Partial<NLU.ModelId>[]>(
+function makeEngineMock(loadedLanguages: string[]): nluCore.Engine {
+  const loadedModels: nluCore.ModelId[] = (<Partial<nluCore.ModelId>[]>(
     loadedLanguages.map(l => ({ languageCode: l }))
-  )) as NLU.ModelId[]
+  )) as nluCore.ModelId[]
```

Comment:
I'm curious why you rename `NLU` to `nluCore`. Except showing a ton of changes to rename the property, I see no reason why you don't keep "NLU", which is much more clear to me.

You can remove the namespace from botpress.d.ts and use it here `import NLU from 'common/nlu-core'`

Reply:
Just because the "NLU" namespace still exists in the SDK. I want to make obvious what is from the SDK from what is from nlu-core.

Reply:
I suggest renaming the NLU namespace in the SDK to something else, since it now represent less than 5% of the codebase, while nluCore represents all the rest.

Reply:
We can't really do that we might break some users that are using NLU namespace for whatever reason ...